corehq/apps/reports/generic.py

```diff
@@ -294,19 +298,29 @@ def timezone(self):
     @property
     @memoized
     def template_base(self):
+        if self.use_bootstrap5:
+            return get_bootstrap5_path(self.base_template)
         return self.base_template
 
     @property
     @memoized
     def template_async_base(self):
+        if self.use_bootstrap5:
+            default_path = "reports/async/bootstrap5/default.html"
+        else:
+            default_path = "reports/async/bootstrap3/default.html"
         if self.asynchronous:
-            return self.base_template_async or "reports/async/bootstrap3/default.html"
+            return self.base_template_async or default_path
         return self.template_base
 
     @property
     @memoized
     def template_report(self):
-        original_template = self.report_template_path or "reports/async/basic.html"
+        if self.use_bootstrap5:
+            template_path = get_bootstrap5_path(self.report_template_path)
+        else:
+            template_path = self.report_template_path
+        original_template = template_path or "reports/async/basic.html"
```

Comment:
Does the `"reports/async/basic.html"` path need to be translated to bootstrap5? Would it make sense to do the bootstrap5 conversion after this?

Reply:
updated:
- https://github.com/dimagi/commcare-hq/pull/35648/commits/dd18ef4f2e8356ef1a5fdb9a948ea7ec6e47a12a
- https://github.com/dimagi/commcare-hq/pull/35648/commits/48201319b467d83f814cf4e3f3eec40a09950237
- https://github.com/dimagi/commcare-hq/pull/35648/commits/b356d1eb09a232c1e97b282a7e639a4bd365bb94